### DISCUSSION TITLE
How to do emotions and podcasts with CLI Inference?

### DISCUSSION THREAD

**Discussion by kamineko16** [SWivid/F5-TTS#211] [Q&A] [answered]
Hi, I can't use gradio_app because of an issue, so my only option right now is CLI Inference. However, when I try to use emotions, it reads the emotions instead of just doing the emotion. For example: "(Angry)Wait!" it says "Angry Wait!" instead.
Also how to do podcasts and emotions in the podcast's?

**Comment by SWivid** [comment]
Check
```
# Multi voice
python inference-cli.py -c samples/story.toml
```
setup in samples/ dir

**Reply by kamineko16** [reply]
> Check
> 
> ```
> # Multi voice
> python inference-cli.py -c samples/story.toml
> ```
> 
> setup in samples/ dir

Thanks! How do you express emotions with the text? I still can't do it, the voice just read it instead...

**Reply by SWivid** [reply]
Not that way you think maybe. The model is to mimic the reference audio. 
If you give a **reference audio** with anger emotion, the model will try to speak in a similar mode.

**Reply by kamineko16** [reply]
> Not that way you think maybe. The model is to mimic the reference audio. If you give a **reference audio** with anger emotion, the model will try to speak in a similar mode.

What do you mean? In the gradio_app interface you can use (Angry) to make it sound angry and so on, I also saw a video example of someone doing the same. Also, how do I choose the speed of the talk as you can in gradio_app? Sometimes it reads the text very fast, and sometimes it reads it regularly.

**Reply by SWivid** [reply, marked as answer]
`(Angry)` means using the reference audio with angry emotion, not just `(Angry)` the word itself!
Check for files under `samples/` you'll see in `story.txt`, the `[country]` indicates using the `country.flac` as reference audio.

speed see gradio_app `advanced setting`.
